src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
```

Comment:
You can use this syntax to add a default value 
```suggestion
export function Sum({ a = 0, b = 0 }) {
```